docs/vector-index/develop/api-reference/index.md

```diff
@@ -71,7 +71,9 @@ Lists all vector indexes.
   <summary>Method response object</summary>
 
 * Success
-    * getIndexNames(): string[]
+    * getIndexes(): VectorIndexInfo[]
```

Comment:
super nit but why not `getIndexes:` instead of `getIndexes():` Just noticed had removed the `()` in below example.

Reply:
`VectorIndexInfo` is a class with accessor methods while `SearchHit` is an interface. I modeled `VectorIndexInfo` after `CacheInfo` which was also a class; but `SearchHit` started as an interface 🤷 . I could add property-like accessors to `VectorIndexInfo`